src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,61 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
+  min-width: $iui-3xl * 2;
+  width: $iui-3xl * 4;
+  max-width: 50vw;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 0 $iui-sm;
+  overflow-x: hidden;
+  overflow-y: auto;
+  overflow-y: overlay;
+  resize: horizontal;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-right: 1px solid t(iui-color-background-5);
+  }
+
+  @include iui-transition-group;
+
+  &-header {
```

Comment:
I am not sure about using this kind of naming. It is a nice feature but it is really hard when you want to find specific class.

Reply:
It's not really hard because the component name is in front of it, so you know where to look.

Reply:
But you can't just copy class name from HTML and search. You will need to first thought why no results are found, then open the SCSS file and search manually line by line.

Reply:
You could just open the correct file and search for the suffix.

I think it's a price worth paying to reduce specificity, which is more important.